src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboard.tsx

```diff
@@ -0,0 +1,60 @@
+import { Outlet, useParams } from "react-router-dom";
+import { GovernanceDAODetails } from "@dex-ui/services";
+import { TokenBalance, useAccountTokenBalances, useDAOs } from "@dex-ui/hooks";
+import { isNil, isNotNil } from "ramda";
+import { Member } from "../MultiSigDAODashboard/types";
+import { DashboardLayout } from "@dex-ui/layouts";
+
+export function GovernanceDAODashboard() {
+  const { accountId: daoAccountId = "" } = useParams();
+  const daosQueryResults = useDAOs<GovernanceDAODetails>(daoAccountId);
+  const { data: daos } = daosQueryResults;
+  const dao = daos?.find((dao) => dao.accountId === daoAccountId);
+
+  const accountTokenBalancesQueryResults = useAccountTokenBalances("");
+  const { data: tokenBalances } = accountTokenBalancesQueryResults;
+
+  const isNotFound = daosQueryResults.isSuccess && isNil(dao);
+  const isDAOFound = daosQueryResults.isSuccess && isNotNil(dao);
+  const isError = daosQueryResults.isError || accountTokenBalancesQueryResults.isError;
+  const isLoading = daosQueryResults.isLoading || accountTokenBalancesQueryResults.isLoading;
+  const errorMessage = daosQueryResults.error?.message || accountTokenBalancesQueryResults.error?.message;
+  const isSuccess = daosQueryResults.isSuccess;
+
+  if (dao) {
+    const { adminId } = dao;
+    const ownerCount = 0;
```

Comment:
we can also add a `TODO` comment for future reference if members are not required in the same page

Reply:
Added a comment for future reference.